classes/MessagedHandler.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace VUMC\REDCapInstrumentAndVariableSQLRenamer;
+
+use REDCap;
+
+class MessagedHandler
+{
+    private $module;
+
+    public $message = '';
+
+    public $messageType = '';
+
+    public $printVariable = '';
+
+    public $printInstrument = '';
+
+    public function __construct(REDCapInstrumentAndVariableSQLRenamer $module)
+    {
+        $this->module = $module;
+    }
+
+    public function messageType($messageType): void
+    {
+        $this->messageType = $messageType;
+    }
+
+    public function addMessage($message): void
```

Comment:
```suggestion
    public function setMessage($message): void
```

You could use "add" here, but when I see the word "add" on a class like this, I expect a 1 to many situation, where I'm adding one to an array.

Reply:
Name changed